### ISSUE TITLE
PrimeIdealsOverPrime(QuadraticField(3), 3) fails wiht this package

### ISSUE THREAD

**Issue by edgarcosta** [rbommel/genus3torsion#1]
Something is off:
```
 echo "AttachSpec(\"Torsion.spec\"); PrimeIdealsOverPrime(QuadraticField(3), 3);" | magma -b 
PrimeIdealsOverPrime(
L: Quadratic Field with defining polynomial $.1^2 - 3 over the ...,
p: 3
)
[/Applications/Magma/package/Ring/Montes/MontesAlgorithm.m:1](
)
In file "/Applications/Magma/package/Ring/Montes/MontesAlgorithm.m", line 290, column 16:
>>   IdealRecord:=recformat<
                  ^
Runtime error in elt< ... >: Expression for field sflPols is not a category or structure

**Comment by rbommel** [comment]
It seems that there might be confusion between my intrinsics List(J::JacCrv) and intrinsic List(Pt::JacCrvPt) and the category List. A quick fix would be to rename these intrinsics.

**Comment by rbommel** [comment]
Should be fixed with commit cb53b7a14f9a5aa60ae442d85b970ddec67c6e0b, but please tell me if something doesn't work.
